crates/host/src/wasmbus/mod.rs

```diff
@@ -840,7 +841,13 @@ pub struct Host {
     host_config: HostConfig,
     host_key: KeyPair,
     labels: HashMap<String, String>,
-    nats: async_nats::Client,
+    ctl_topic_prefix: String,
+    /// NATS client to use for control interface subscriptions and jetstream queries
+    ctl_nats: async_nats::Client,
+    /// NATS client to use for actor RPC calls
+    rpc_nats: async_nats::Client,
+    /// NATS client to use for communicating with capability providers
+    prov_rpc_nats: async_nats::Client,
```

Comment:
Nit: these (almost?) sorted before

Reply:
I just put them in this order because that's the order I put em in everywhere else (assuming you're asking for ctl -> prov RPC -> RPC)